File: 비주얼 베이직/2019-1-간단한프로그램2-김유빈-pp71/2019-1-간단한프로그램2-김유빈-pp71/Form3.vb
Public Class Form3

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim N, i, oddsum, evensum As Integer
        N = TextBox1.Text
        TextBox2.Text = ""
        TextBox3.Text = ""
        For i = 1 To N Step 1
            If (i Mod 2 = 1) Then

            End If
            oddsum = oddsum + i
            If (TextBox2.Text = "") Then
                TextBox2.Text = TextBox2.Text & i
            Else
                TextBox2.Text = TextBox2.Text & " +" & i
            End If

            Else
            evensum = evensum = i
            If (TextBox3.Text = "") Then
                TextBox3.Text = TextBox3.Text & i
            Else
                TextBox3.Text = TextBox3.Text & " +" & i
            End If

        Next
        TextBox2.Text = TextBox2.Text & "=" & oddsum
        TextBox3.Text = TextBox3.Text & "=" & evensum

    End Sub


End Class

File: C#/2019-2-C샵-2주차-두숫자의연산-김유빈/2019-2-C샵-2주차-두숫자의연산-김유빈/Form3.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _2019_2_C샵_2주차_두숫자의연산_김유빈
{
    public partial class Form3 : Form
    {
        public Form3()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
           
            long Sum1 = 0;
            long Sum2 = 0;
            int num, i;

            num = int.Parse(textBox1.Text);
            textBox2.Text = "0";
            textBox3.Text = "0";

            for (i = 1; i <= num; ++i)
            
                if (i % 2 == 1)
                {
                    Sum2 = Sum2 + i;
                    textBox2.Text = textBox2.Text  + " + " + i;
                }



                else if (i % 2 == 0)
                {
                    Sum1 = Sum1 + i;
                    textBox3.Text = textBox3.Text + " + "+ i ;

                };
                textBox3.Text = textBox3.Text + " = " + Sum1;
                textBox2.Text = textBox2.Text + " = " + Sum2;
            }
        }
    }

